--- a/_footer.php	2025-12-04 07:14:56.000000000 +1000
+++ b/_footer.php	2026-08-04 06:43:58.240355668 +1000
@@ -16,5 +16,8 @@
       data-read-less-text="<?php echo __('Read less'); ?>">
     </div>
   </div>
-
+  <?php $uqComp = sfConfig::get('app_uq_reusable_components', ''); ?>
+  <?php if (!empty($uqComp)) { ?>
+    <script type="text/javascript" defer="defer" src="<?php echo $uqComp; ?>"></script>
+  <?php } ?>
 </footer>
